sympy/core/add.py

```diff
@@ -594,10 +594,12 @@ def _eval_is_positive(self):
     def _eval_is_nonnegative(self):
         from sympy.core.exprtools import _monotonic_sign
         if not self.is_number:
-            c, a = self.as_coeff_Add()
-            if not c.is_zero and a.is_nonnegative:
+            c, a = self.as_independent(*self.free_symbols)
```

Comment:
@smichr Actually, I deleted sympy repo from my leptop and Currently, i am doing web development internship so i have no time for this. ```sorry for that```